edx_notifications/stores/sql/models.py

```diff
@@ -231,7 +233,56 @@ class Meta(object):
         db_table = 'edx_notifications_notificationchannel'
 
 
-class SQLUserNotificationPreferences(models.Model):
+class SQLNotificationPreference(models.Model):
+    """
+    Notification preference
+    """
+    class Meta(object):
+        """
+        ORM metadata about this class
+        """
+        app_label = 'edx_notifications'  # since we have this models.py file not in the root app directory
+        db_table = 'edx_notifications_notificationpreference'
+
+    # the internal name is the primary key
+    name = models.CharField(primary_key=True, max_length=255)
+
+    display_name = models.CharField(max_length=255)
+
+    display_description = models.CharField(max_length=1023)
```

Comment:
We're going to need a 'default_value' column as well: charfield(max_length=255, null=True)